torch/_dynamo/eval_frame.py

```diff
@@ -643,7 +643,10 @@ def output(self, target, args, kwargs):
 
         def run_node(self, n):
             self.current_node = n
-            return super().run_node(n)
+            r = super().run_node(n)
+            if "val" in self.current_node.meta:
+                r.node.meta["val"] = self.current_node.meta["val"]
```

Comment:
Hmm, how did we lost it in the first place?

Reply:
the fx Transformer makes a new graph, and nothing gets propagated unless you do it explicitly.